package.json

```
@@ -46,8 +46,8 @@
     "eslint-plugin-react": "^7.12.4",
     "eslint-plugin-standard": "^4.0.0",
     "file-loader": "^1.1.6",
-    "flow-bin": "^0.100.0",
-    "flow-typed": "^2.5.2",
+    "flow-bin": "^0.144.0",
+    "flow-typed": "^3.2.1",
```

`yarn flow:install`,  fails with error:
```
UNCAUGHT ERROR: Error: Invalid npm libdef version! It appears to be a non-continugous range.
    at pkgVersionMatch (/opt/app-root/src/project/node_modules/flow-typed/dist/lib/npm/npmLibDefs.js:256:11)
    at defs.filter.def (/opt/app-root/src/project/node_modules/flow-typed/dist/lib/npm/npmLibDefs.js:281:82)
    at Array.filter (<anonymous>)
    at filterLibDefs (/opt/app-root/src/project/node_modules/flow-typed/dist/lib/npm/npmLibDefs.js:275:15)
    at findNpmLibDef (/opt/app-root/src/project/node_modules/flow-typed/dist/lib/npm/npmLibDefs.js:329:27)
error Command failed with exit code 1.
```

Updating to 3.3.1 fix it.
```suggestion
    "flow-typed": "^3.3.1",
```

---

Yes, it's pointed out in the description